src/main/scala/org/clulab/timenorm/scate/Types.scala

```diff
@@ -824,6 +824,13 @@ case class IntersectionRI(repeatingIntervals: Set[RepeatingInterval],
   override val base: TemporalUnit = repeatingIntervals.minBy(_.base.getDuration).base
   override val range: TemporalUnit = repeatingIntervals.maxBy(_.range.getDuration).range
 
+  for {
```

Comment:
Please add a comment here reminding us why we need this code in the constructor.